hack/generate-manifest.sh

```diff
@@ -154,9 +158,13 @@ case $key in
     SRIOV=true
     shift
     ;;
-    --flexible-ipam)
-    FLEXIBLE_IPAM=true
-    shift
+    --secondary-bridge)
+    SECONDARY_BRIDGE="$2"
+    shift 2
+    ;;
+    --physical-interface)
+    PHYSICAL_INTERFACE="$2"
+    shift 2
```

Comment:
I see that you switched to using Helm directly in the end. Should these new flags be removed?

Reply:
I like to keep them as it is still simpler to run generate-manifest.sh in some cases.